ui/pages/login/index.js

```diff
@@ -220,11 +239,7 @@ export default function Login() {
         <button className='mt-4 mb-2 flex w-full cursor-pointer justify-center rounded-md border border-transparent bg-blue-500 py-2 px-4 text-sm font-medium text-white shadow-sm hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2'>
           Log in
         </button>
-        {error && (
-          <p className='absolute -bottom-3.5 mx-auto w-full text-center text-2xs text-red-500'>
-            {error}
-          </p>
-        )}
+        {error && <ErrorMessage message={error} />}
```

Comment:
Would recommend using a <p> tag here as we need to align the error message nicely with the font size and spacing, and to keep it obvious as to what's being rendered.